src/main/java/racingcar/RandomNum.java

```diff
@@ -0,0 +1,12 @@
+package racingcar;
+
+import java.util.Random;
+
+public class RandomNum {
```

Comment:
약어나 축약형 보다는, 완전한 단어로 작성하여 의미를 분명하게 표현해보는 건 어떨까요?
```suggestion
public class RandomNumber {
```